values.md

```diff
@@ -1,4 +1,10 @@
-# Kubernetes Community Values
```

Comment:
Doh, I missed this when looking at it yesterday. Like the mentoring doc, we still want to have this as it lives in both places =/

Reply:
Soz! I'm reverting the deletion!